app/models/realtime_data/sip_options_prober.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+class RealtimeData::SipOptionsProber < YetiResource
+  include ActiveModel::Validations
+  include WithQueryBuilder
+
+  class << self
+    def query_builder_find(id, **_)
+      node_id, id = id.split('*')
+      node = Node.find(node_id)
+      result = NodeRpcClient.perform_parallel([node], default: []) do |client, node|
+        result = client.sip_options_probers([id])
+        result.map { |row| row.merge(node: node) }
+      end
```

Comment:
we don't need to perform parallel operation for one node

just build client by node and get data from it